File: Ejercicios/Java/Ejercicio9.java
import java.util.LinkedList;
public class Ejercicio9 {
    public static void main(String[] args) {
        LinkedList<String> turnos = new LinkedList<>();
        turnos.addLast("Turno A");
        turnos.addLast("Turno B");
        System.out.println("Atendiendo: " + turnos.removeFirst());
        System.out.println("Siguiente en espera: " + turnos.getFirst());
    }
}

File: Ejercicios/C++/Ejercicio9.cpp
#include <iostream>
#include <list>
using namespace std;
int main() {
    list<string> turnos;
    turnos.push_back("Turno A"); turnos.push_back("Turno B");
    cout << "Atendiendo: " << turnos.front() << "\n";
    turnos.pop_front(); 
    cout << "Siguiente: " << turnos.front() << "\n";
    return 0;
}
